src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsUcCaseData.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SscsUcCaseData {
+
+    private List<String> ucWriteFinalDecisionPhysicalDisabilitiesQuestion;
+    private List<String> ucWriteFinalDecisionMentalAssessmentQuestion;
+    private String ucWriteFinalDecisionMobilisingUnaidedQuestion;
+    private String ucWriteFinalDecisionStandingAndSittingQuestion;
+    private String ucWriteFinalDecisionReachingQuestion;
+    private String ucWriteFinalDecisionPickingUpQuestion;
+    private String ucWriteFinalDecisionManualDexterityQuestion;
+    private String ucWriteFinalDecisionMakingSelfUnderstoodQuestion;
+    private String ucWriteFinalDecisionCommunicationQuestion;
+    private String ucWriteFinalDecisionNavigationQuestion;
+    private String ucWriteFinalDecisionLossOfControlQuestion;
+    private String ucWriteFinalDecisionConsciousnessQuestion;
+    private String ucWriteFinalDecisionLearningTasksQuestion;
+    private String ucWriteFinalDecisionAwarenessOfHazardsQuestion;
+    private String ucWriteFinalDecisionPersonalActionQuestion;
+    private String ucWriteFinalDecisionCopingWithChangeQuestion;
+    private String ucWriteFinalDecisionGettingAboutQuestion;
+    private String ucWriteFinalDecisionSocialEngagementQuestion;
+    private String ucWriteFinalDecisionAppropriatenessOfBehaviourQuestion;
+    private String ucWriteFinalDecisionSchedule7ActivitiesApply;
+    private List<String> ucWriteFinalDecisionSchedule7ActivitiesQuestion;
+    private String dwpReassessTheAward;
```

Comment:
there are some few missing fields such as 
lcwaAppeal,
showSchedule7ActivitiesPage, 
showSchedule8Paragraph4Page,
doesSchedule8Paragraph4Apply,
these were added to the corresponding CCD PR https://github.com/hmcts/sscs-ccd-definitions/pull/452/files